src/options.test.js

```diff
@@ -378,6 +378,21 @@ describe('options', () => {
     })
   })
 
+  describe('send args as option', () => {
+    it('should send args to jq execution', done => {
```

Comment:
Can you change to the title to refer to `--argjson`?

Reply:
Can you suggest an alternative. I am using the args object to determine if we need `--arg` or `--argjson` flag. 

If we pass,
```
{
 arg: { A: "string value" }
}
```
it will pass `--arg A "string value"`

If we pass,
```
{
 arg: { A: [JSON_OBJECT] }
}
```
it will pass `--argjson A "stringified JSON_OBJECT"`

My way, we can even pass,
```
{
 arg: { A: "string value", B: [JSON_OBJECT] }
}
```
and it will pass `--arg A "string value" --argjson B "stringified JSON_OBJECT"`

passing

```
{
 argjson: { A: [JSON_OBJECT] }
}
```
seemed counter intuitive while we have a similar `arg` object property.

Reply:
I renamed the test cases along with a few more tests to make things clear.